spring-xd-spark-streaming-tests/src/test/java/org/springframework/xd/spark/streaming/RabbitTransportSparkStreamingTests.java

```diff
@@ -47,9 +47,8 @@ protected void createStream(String streamName, String stream) {
 		addQueueNames(streamName, stream);
 	}
 
-	@After
-	public void cleanupRabbitQueues() {
-		super.tearDown();
+	@AfterClass
```

Comment:
Don't agree with this: what if there are 2 tests that use the same queue names. Should be cleared between tests, not at the end of the "suite"